alacritty/src/event.rs

```diff
@@ -923,6 +932,29 @@ impl<N: Notify + OnResize> Processor<N> {
                     },
                     TerminalEvent::MouseCursorDirty => processor.reset_mouse_cursor(),
                     TerminalEvent::Exit => (),
+                    TerminalEvent::CursorStartBlinking => {
+                        let rate = processor.ctx.config.cursor.blink_rate;
+                        let scheduler = &mut processor.ctx.scheduler;
+                        if rate != 0 && scheduler.get_mut(TimerId::CursorBlinking).is_none() {
+                            processor.ctx.scheduler.schedule(
+                                GlutinEvent::UserEvent(Event::TerminalEvent(
+                                    TerminalEvent::CursorBlink,
+                                )),
+                                Duration::from_millis(rate),
+                                true,
+                                TimerId::CursorBlinking,
+                            )
+                        }
+                    },
+                    TerminalEvent::CursorBlink => {
+                        processor.ctx.terminal.cursor_blinking_out ^= true; // flip boolean value
```

Comment:
```suggestion
                        processor.ctx.terminal.cursor_blinking_out ^= true;
```

This comment is pointless.